signer/src/config/mod.rs

```diff
@@ -38,6 +38,10 @@ pub const MAX_BITCOIN_PROCESSING_DELAY_SECONDS: u64 = 300;
 /// Maximum configurable delay (in seconds) before processing new SBTC requests.
 pub const MAX_REQUESTS_PROCESSING_DELAY_SECONDS: u64 = 300;
 
+/// Maximum amount of signers supported by our smart contracts
+/// See https://github.com/stacks-sbtc/sbtc/issues/1694
```

Comment:
nit: I would refer to 1620 here for the actual issue.

Reply:
Imo after reading plain 1620 I would have no idea how this related to this PR. Even with clarification that only 4th point related -- better but still I don't really like explanation, so I'd point to issue of this PR and maybe sometime somebody will post explanation under the issue

Reply:
My (very shallow) fear is that we are linking to an issue that will be closed (and forgotten) after this is merged, and it's not the one tracking the underlying issue that we need to solve, so we could avoid this indirection and just point to the actual issue. Although it's true that 1620 contains *a lot* of sutff, so pointing out to the specific point would def help.

Reply:
Hmm, I don't think there is anything bad refering to closed issues, they are closed, not deleted, and if from issue description it would be unclear why 16 the one for whom it will be unclear will post updates (even to closed issue). And also this one contains link to 1620 so it provide not less informaiton

Reply:
I would keep the in-code issue linked to #1694 (which itself links to #1620) because it gives slightly more context, and update point 4 in #1620 to include a link to #1694 (or add it as a sub-task) which will show its status as completed.